src/item.cpp

```diff
@@ -3043,14 +3154,17 @@ void item::combat_info( std::vector<iteminfo> &info, const iteminfo_query *parts
         if( parts->test( iteminfo_parts::BASE_MOVES ) ) {
             info.push_back( iteminfo( "BASE", _( "Moves per attack: " ), "",
                                       iteminfo::lower_is_better, attack_time() ) );
-            double dps = ( dmg_bash + dmg_cut + dmg_stab ) * to_moves<int>( 1_seconds ) /
-                         static_cast<double>( attack_time() );
-            static const matec_id rapid_strike( "RAPID" );
-            if( has_technique( rapid_strike ) ) {
-                dps *= 100.0 / 66;
-            }
-            info.push_back( iteminfo( "BASE", _( "Damage per second: " ), "",
-                                      iteminfo::is_decimal, dps ) );
+            info.emplace_back( "BASE", _( "Typical damage per second: " ), "" );
```

Comment:
If you can find a succinct way to express it, I think it would help avoid confusion to clarify that these values are:
* Specific to current circumstances (e.g. encumbrance, temporary stat changes) and thus only meaningful if the avatar is currently in a "combat-ready state".
* Random, and thus likely to differ slightly on each observation.

Reply:
I tried to express that in the help file.  There's not enough space in the info planel to put in a long explanation and caveats.